### DISCUSSION TITLE
erro ao instalar no delphi 11.3 windows 11

### DISCUSSION THREAD

**Discussion by Ronierys2** [project-jedi/jcl#134] [Q&A]
![image](https://github.com/project-jedi/jcl/assets/53917704/3ba02479-cb14-4168-a5e3-645f289b3722)
ao executar o bat de instalação tenho a seguinte erro descrito na imagem acima. instalei vários outros componentes e somente com o jedi estou com esse erro. alguém pode me ajudar por favor.

[inglês]
when running the installation bat I have the following error described in the image above. I installed several other components and only with jedi I have this error. Can someone help me please.

**Comment by ronaldhoek** [comment]
Please post in English (or add English translation to the question)

**Comment by megamurmulis** [comment]
Either BPL is not in PATH
or perhaps PATH is too long and could not find bpl?

**Comment by JuarezBurda** [comment]
Ronierys, como você resolveu este problema? Estou obtendo a mesma mensagem. Já reinstalei o RAD Studio e continua dando o mesmo erro.

**Reply by Ronierys2** [reply]
formatei o computador e instalei tudo de novo do ZERO.

[inglês]
I formatted the computer and installed everything again from scratch

**Reply by megamurmulis** [reply]
Is this issue still happening to you?
~ given that `35.0` = `Delphi 11 Alexandria`, not `10.5` => incorrect detection in the installer - so perhaps using outdated JCL or maybe did not copy updated 2 JEDI .inc files in `jcl\source\include\jedi`?
